tests/events/propagation.cpp

```diff
@@ -637,6 +647,110 @@ void EventPropagationTestCase::DocView()
 #endif // wxUSE_TOOLBAR
 }
 
+#if wxUSE_AUI
+    void EventPropagationTestCase::DocViewAui()
```

Comment:
Trivial, but could you please remove the indent of this function?

Reply:
Done.  See https://github.com/wxWidgets/wxWidgets/pull/25336/commits/12a20d786492e998d4518ac7d8ab21eee270ad90